src/cio_websocket.c

```diff
@@ -447,6 +445,17 @@ static void handle_close_frame(struct cio_websocket *ws, uint8_t *data, uint_fas
 	}
 }
 
+static void pong_frame_written(struct cio_websocket *ws, void *handler_context, enum cio_error err)
+{
+	(void)handler_context;
+
+	if (cio_likely(err == CIO_SUCCESS)) {
+		ws->ws_private.bs->read_exactly(ws->ws_private.bs, ws->ws_private.rb, 1, get_header, ws);
+	} else {
+		// TODO
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=c%3AS1135)